apps/frontend/hub/src/lib/server/api/tests/health_db.test.ts

```
@@ -0,0 +1,96 @@
+// apps/frontend/hub/src/lib/server/api/tests/health_db.test.ts
+//
+// C-436 AC-4: DB health reports the binding.
+//
+// Tests three states: healthy D1 (ok with roundTripMs), absent binding
+// (unconfigured), and throwing D1 (unreachable). Verifies the response
+// never contains a credential, connection string, or internal identifier.
+
+import { afterAll, beforeAll, describe, expect, mock, test } from 'bun:test';
```

Biome (2.5.7)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)